.github/workflows/ci.yml

```
@@ -0,0 +1,10 @@
+name: ci
+on: [push]
+jobs:
+  ci:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+      - run: yarn
```

Just to be safe, let's add `--ignore-scripts` here to prevent 3rd party dependencies from running malicious post-install scripts that could steal our secrets
```suggestion
      - run: yarn install --ignore-scripts
```